transport/server.cc

```diff
@@ -619,7 +619,7 @@ void cql_server::connection::on_connection_close()
 std::tuple<net::inet_address, int, client_type> cql_server::connection::make_client_key(const service::client_state& cli_state) {
     return std::make_tuple(cli_state.get_client_address().addr(),
             cli_state.get_client_port(),
-            cli_state.is_thrift() ? client_type::thrift : client_type::cql);
+            client_type::cql);
```

Comment:
Maybe you can get rid of this parameter completely?

Reply:
i will implement it in another commit in this PR. the first commit is supposed to be as silly / conservative as possible.